src/couch_replicator/src/couch_replicator_job.erl

```diff
@@ -865,6 +948,11 @@ state_strip_creds(#rep_state{source = Source, target = Target} = State) ->
 
 
 adjust_maxconn(Src = #{<<"http_connections">> := 1}, RepId) ->
+    ?LOG_NOTICE(#{
```

Comment:
The original message logged RepId, should we include it in the logger report too?

Reply:
Good catch, fixed.